### DISCUSSION TITLE
docker-compose.yaml issue

### DISCUSSION THREAD

**Discussion by DebabrataTah** [apache/airflow#28150] [General]
### Apache Airflow version

2.5.0

### What happened

I tried to install airflow localhost. I downloaded the docker-compose.yaml file from the link "https://airflow.apache.org/docs/apache-airflow/stable/docker-compose.yaml". When I ran the command "docker-compose up airflow-init" or the command "docker-compose up", it is giving error like " **Invalid interpolation format for "environment" option in service "flower": "${_PIP_ADDITIONAL_REQUIREMENTS:-}"** ".

### Operating System

Microsoft Windows 10 Enterprise

### Deployment

Docker-Compose

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by xlanor** [comment]
What version of docker-compose are you using?

**Reply by DebabrataTah** [reply]
docker-compose version 1.11.2

**Comment by potiuk** [comment]
You have an old docker-compose version installed. See prerequisites:

https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html#before-you-begin
